### ISSUE TITLE
About the training data

### ISSUE THREAD

**Issue by zjYao36** [bytetriper/LM4LV#1]
Very interesting work! May I ask, 
1. during the training of LM4LV, do I need to generate the degradation image of the corresponding task in advance. In other words, I need to generate 595k low-quality images before training. 
2. Another issue is the choice of dataset, as far as I know LLAVA595k is not considered a HQ dataset, why choose it as the training set and not others, such as imagenet.
3. Thank you for your answers and for your discussion

**Comment by bytetriper** [comment]
Hi Zhengjian, thanks for your interest!

1.
Yes, the degradation needs to be added before training. This is kind of dumb but is required by the current data format. I may add a image-specific transformation to enable data-aug and runtime degradation later.

2.
Yes I choose LLAVA595k for an alignment to previous MLLM works (especially VL tasks). For the same reason the test set in the paper is NoCaps , which is not a common choice for vision-only tasks. You can change it to arbitrary image dataset you want. I think our method generalizes beyond LLAVA595k.

**Comment by zjYao36** [comment]
> Hi Zhengjian, thanks for your interest!
> 
> 
> Yes, the degradation needs to be added before training. This is kind of dumb but is required by the current data format. I may add a image-specific transformation to enable data-aug and runtime degradation later.
> 
> 
> Yes I choose LLAVA595k for an alignment to previous MLLM works (especially VL tasks). For the same reason the test set in the paper is NoCaps , which is not a common choice for vision-only tasks. You can change it to arbitrary image dataset you want. I think our method generalizes beyond LLAVA595k.

Thank you very much for your reply, very helpful!
